docs-src/cloud-context/namespaces-create.md

```diff
@@ -46,7 +46,7 @@ To create a Namespace in Temporal Cloud, gather the following information:
 1. In **Retention Period**, specify a value from 1 to 90 days.
    When choosing this value, consider your needs for Event History versus the cost of maintaining that Event History.
    Typically, a development Namespace has a short retention period and a production Namespace has a longer retention period.
```

Comment:
I don't think we distinguish between this, and all namespaces have a default retention period when they are created. @antmendoza can you confirm please?

Reply:
When you use the UI to create a Namespace, the slider for Retention Period defaults to 30 days.

If we need to update this section, let's create a separate task.

Reply:
@tsurdilo as @djmagee states, the default value in the form is 30 days.